community/server/src/test/java/org/neo4j/server/rest/PathsDocIT.java

```diff
@@ -202,8 +242,6 @@ public void shouldGetCorrectDijkstraPathsWithWeights() throws Exception
         assertEquals( 1.5, secondPath.get( "weight" ) );
         // 5 = 3 +2
         assertEquals( 5, (Integer) firstPath.get( "length" ) + (Integer) secondPath.get( "length" ) );
-        assertThatPathHasLength( firstPath, 3 );
-        assertThatPathHasLength( secondPath, 2 );
```

Comment:
If that is the case then something is seriously broken, but yeah, better safe than sorry I suppose.
Fixed.